### ISSUE TITLE
Tests pass even when they should fail

### ISSUE THREAD

**Issue by noamraph** [rust-lang/rustlings#152]
In tests1.rs I put `assert!(false)` and in tests2.rs I put `assert_eq!(2, 3)` and I get:

```
> rustlings run --test exercises/tests/tests1.rs
✓ Successfully tested exercises/tests/tests1.rs!

> rustlings verify
...
✓ Successfully tested exercises/tests/tests1.rs!
✓ Successfully tested exercises/tests/tests2.rs!
```

It seems that the only requirement is for the test to compile, regardless of its result.

**Comment by Tyilo** [comment]
Duplicate of #149

**Comment by komaeda** [comment]
Closed by 35c3d0b, will release a bugfix release right now.

**Comment by komaeda** [comment]
Released [1.2.2](https://github.com/rust-lang/rustlings/releases/tag/1.2.2)
